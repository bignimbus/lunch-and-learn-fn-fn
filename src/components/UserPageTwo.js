import React, { Component } from 'react';
import PropTypes from 'prop-types';
import withUser from '../hocs/withUser';

export const UserPageTwo = ({ user }) => (
  <h1>{ user.username }'s Photos</h1>
);

UserPageTwo.propTypes = {
  user: PropTypes.shape({
    username: PropTypes.string.isRequired,
  }).isRequired,
};

const UserPageTwoContainer = withUser(UserPageTwo);

export default UserPageTwoContainer;
