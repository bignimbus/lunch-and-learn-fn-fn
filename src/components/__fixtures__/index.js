export const user = {
  username: 'foo',
};
