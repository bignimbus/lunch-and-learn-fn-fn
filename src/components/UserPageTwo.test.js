import React from 'react';
import renderer from 'react-test-renderer';
import { UserPageTwo } from './UserPageTwo';
import { user } from './__fixtures__';

const renderComponent = () => renderer.create(
  <UserPageTwo user={user} />,
);


describe('UserPageTwo', () => {
  test('#render', () => {
    const component = renderComponent();
    expect(component.toJSON()).toMatchSnapshot();
  });
});
