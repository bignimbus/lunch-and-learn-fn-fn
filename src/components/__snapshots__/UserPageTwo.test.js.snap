// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPageTwo #render 1`] = `
<h1>
  foo
  's Photos
</h1>
`;
